Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -170,46 +170,51 @@ class BrowserCoordinator: BaseCoordinator,
             return false
         }
 
+        switch route {
+        case .searchQuery, .search, .searchURL, .glean, .homepanel, .action, .fxaSignIn, .defaultBrowser:
```

Comment:
I think it misses the `.settings` route?

Reply:
It's right below this, that one had some custom logic so it's the one route type which needs to check the associated value first (afaics).

Reply:
Ah oups 🙈